### ISSUE TITLE
"Error: (200)OK" when trying to add a movie

### ISSUE THREAD

**Issue by tmortagne** [roboticsound/Pulsarr#43]
When I click the icon everything looks OK (the popup contains the right path in "Folder" so it can definitely access the Radarr instance data properly) but when I choose the profile and click "Add" I end up with "Error: (200)OK" (which is a pretty weird error from HTTP point of view :)).

Environment:
* Radarr 0.2.0.995 with form based authentication
* Firefox 58.0.2
* Ubuntu 17.10 64b

**Comment by roboticsound** [comment]
Yes you are right, that is a strange error! Does it happen on all websites, movies/series etc? So far I am unable to reproduce the error.

**Comment by tmortagne** [comment]
I only tried on http://www.imdb.com/title/tt1365519/.

**Comment by andrewrmoore** [comment]
Seeing this too. With movies and TV shows, any of them.

@tmortagne are you using a reverse proxy or is it direct?

**Comment by roboticsound** [comment]
@andrewrmoore what is your setup?

**Comment by andrewrmoore** [comment]
Radarr 0.2.0.995
Sonarr 2.0.0.5153
Pulsarr 2.1 running on Chrome 64.0.3282.186

Radarr and Sonarr are accessed via Caddy reverse proxy. All running on CentOS 7.4

Pulsarr does connect, as in it picks up the Profiles and Folders but fails with
> Error: (200)

when adding a movie/TV show.

**Comment by tmortagne** [comment]
> @tmortagne are you using a reverse proxy or is it direct?

Apache httpd reverse proxy indeed. You think it could be related ?

**Comment by tmortagne** [comment]
Here is my conf it it can help:

```
<VirtualHost *:443>
  ...ssl stuff...

  ServerName radarr.myhost.org
  ServerSignature Off

  ProxyPreserveHost On

  DocumentRoot /var/www

  ProxyPass / http://127.0.0.1:7878/
  ProxyPassReverse / http://127.0.0.1:7878/

  ...log stuff...
</VirtualHost>
```

**Comment by evulhotdog** [comment]
So I was actually having the same issue trying to access my Radrarr installation.

I was able to add it as this screenshot, but got the same Error(200) as others were getting. 
![image](https://user-images.githubusercontent.com/365456/37567516-9767dff0-2a9e-11e8-9df5-6180e39523ba.png)

That being said, my actual Radarr installation is at path is: http://nas-7010:7878/radarr/

However, when I try to add "http://nas-7010:7878/radarr/", WITH the port configured as 7878 it doesn't work and the extension just hangs on save. But if I only use the URL (including the base path and porth), and leaving the port configuration blank, it works perfect:
![image](https://user-images.githubusercontent.com/365456/37567584-83159bea-2a9f-11e8-8e56-26d0944c928d.png)


I would try not using the port in the dedicated port field, but include the port and base path in the URL and see if that works for you.

I also have another server that I use, that is behind the LSIO letsencrypt container, which is just at server.domain.com/sonarr/ and it works perfectly with this configuration. Again, leaving the port blank, but letting it assume port 443 via https.
![image](https://user-images.githubusercontent.com/365456/37567611-c9e34f72-2a9f-11e8-9129-8e7299c6a942.png)

**Comment by roboticsound** [comment]
Hi @evulhotdog ,

Thanks for the great post (sorry I've been unwell and couldn't reply sooner). Yes this has been mentioned before and is something I will improve in the UI in the future. @tmortagne and @andrewrmoore were you able to get things working using the above setup?

**Comment by tmortagne** [comment]
I actually made a mistake in my configuration but I did not noticed it because it seemed to be working: my radarr is behind `https` but I entered `http`. The reason is was half working is because there is an automatic redirect from `http` to `https`.

So the reason the reported error was so weird is most probably because of the redirect.

Sorry for the noise...

**Comment by Virtike** [comment]
> So I was actually having the same issue trying to access my Radarr installation.

> (...) But if I only use the URL (including the base path and port), and leaving the port configuration blank, it works perfect ( http://192.168.X.X:7878/radarr/ ) 

@evulhotdog Thank you. I was getting the "Error: (200)OK" & couldn't add movies, changed the config to include the port in the path as above with the actual port field blank, and it now works probably - can add movies, check if already monitored etc.
